functions.php

```diff
@@ -45,15 +49,42 @@ function sendData($db, $getSeason, $getPremier, $getRunnersUp, $getScore) {
  * @param array $finals
  * @return string
  */
-function getTeamData(array $finals): string {
+function getTeamData(array $finals): string 
+{
     $team = '';
     foreach($finals as $final) {
         if ($final === []) {
             return 'Got data?';
         } else {
-            $team .= '<div class="winners-box">' .
-                        '<img src="'.$final['Image'].'" alt="Grandfinal team photo" class="winners-image">' . '<div class="match-stats"><h6 class="team-name">' . $final['Premier'] . '</h6><p>' . $final['Season'] . '</p><p>' . $final['Runner-Up'] . '</p><p>' . $final['Score'] . '</p></div></div>';
+            if($final['Delete'] == '0') {
+                $team .= '<div class="winners-box">' .
+                        '<img src="'.$final['Image'].'" alt="Grandfinal team photo" class="winners-image">' . '<div class="match-stats"><h6 class="team-name">' . $final['Premier'] . '</h6><p>' . $final['Season'] . '</p><p>' . $final['Runner-Up'] . '</p><p>' . $final['Score'] . '</p></div>' . '<form action="delete.php" method="POST"><input type="submit" value="Delete"/>' . '<form class="edit" action="edit.php" method="POST"><input type="submit" value="Edit"/><input class="hidden" name="dataID" value="' . $final['id'] . '"></form></div>';
+            }   
         }
     } 
     return $team;
+}
+
+/**
+ * Targets the ID in the database
+ *
+ * @return void
+ */
+    function targetID()
+    {
+            return $_POST['dataID'];
+    }
```

Comment:
incorrect indentation and missing return type

Reply:
indentation corrected
return type set to int